src/app/pages/FastBtcPage/components/Withdraw/WithdrawDetails.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useContext, useMemo } from 'react';
+import { useTranslation } from 'react-i18next';
+import { AssetSymbolRenderer } from 'app/components/AssetSymbolRenderer';
+import { translations } from 'locales/i18n';
+import { WithdrawContext } from '../../contexts/withdraw-context';
+import { toNumberFormat } from '../../../../../utils/display-text/format';
+import { LoadableValue } from '../../../../components/LoadableValue';
+import { btcInSatoshis } from 'app/constants';
+import { NetworkAwareComponentProps } from '../../types';
+import { getBTCAssetForNetwork } from '../../helpers';
+
+const DYNAMIC_FEE_DIVISOR = 10000;
```

Comment:
Add a comment here to indicate why this value is used

Reply:
Refactored this into constants file and added comment linking to github repo where this value is pulled from